spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -305,6 +305,27 @@ that it could not until explicitly told that it can clean up by providing a meth
 
 If multiple methods are annotated with the same annotation an arbitrary one will be chosen.
 
+[[config-parameters]]
+==== Configuration parameters
+
+This specification has 1 configuration parameter which is retrieved using the MicroProfile Config principals.
+
+     mp.lra.LRA.value=SUPPORTS
+
+The allowed values for the configuration parameter are `SUPPORTS`, the default value, and `NOT_SUPPORTED`.
```

Comment:
so if we allow only SUPPORTS/NOT_SUPPORTED wouldn't it be better to have only a boolean value mp.lra.propagation.active=true/false? Won't this lead developers to define all possible LRA.Type values?

Reply:
+1

Reply:
yes agree. I proposed the SUPPORTS/NOT_SUPPORTED since we only discussed based on LRA.Type